assets/elm/src/PathView.elm

```diff
@@ -36,10 +38,38 @@ viewPath { done, current, todo } =
         ]
 
 
-todoHtml : List Lesson -> Html a
+todoHtml : List Lesson -> Html Msg
 todoHtml todos =
-    Html.div [ class "mypath-todo" ] (List.map (SmallCard.view Todo) todos)
-
+    Html.div [ class "path-layout" ]
+             [ Html.div [ class "mypath-todo" ] (allLessons todos) ]
+
+allLessons : List Lesson -> List (Html Msg)
+allLessons listLessons =
+  let
+      firstLesson =
+        listLessons
+        |> List.head
+        |> Maybe.map firstLessonWrapper
+
+      restLessons =
+        listLessons
+        |> List.tail
+        |> Maybe.map (List.map lessonWrapper)
+
+      allLessons =
+        Maybe.map2 (::) firstLesson restLessons
+  in
+      Maybe.withDefault [] allLessons
+
+firstLessonWrapper : Lesson -> Html Msg
```

Comment:
firstLessonWrapper and lessonWrapper share a lot of code, but wasn't sure how best to do this, so left with duplication.